### DISCUSSION TITLE
How to get a redis list length

### DISCUSSION THREAD

**Discussion by hansongChina** [kedacore/keda#3991] [Q&A / Need Help] [answered]
When using the redis sentinel list model, how to actively get the length of the list monitored. Easy to show to the user, know the current list of extrusion volume

**Comment by kubroid** [comment]
If I got question right:

from redis-cli: llen <list-name>
from kubectl: kubectl get hpa

**Comment by hansongChina** [comment, marked as answer]
i get it metrics server
https://keda.sh/docs/2.12/operate/metrics-server/
